src/ansible_navigator/cli.py

```diff
@@ -30,9 +31,9 @@
 
 
 try:
```

Comment:
Maybe we need something like
```suggestion
__version__: Union[Constants, str]
try:
```
?

Reply:
yeah, this is a better way to handle it on the ultimate assignment